src/models/disassemblyoutput.cpp

```diff
@@ -25,34 +25,52 @@
   along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#include "data.h"
 #include "disassemblyoutput.h"
-#include <QStandardPaths>
+#include "data.h"
 #include <QApplication>
-#include <QProcess>
 #include <QDebug>
+#include <QProcess>
+#include <QRegularExpression>
+#include <QStandardPaths>
 
 static QVector<DisassemblyOutput::DisassemblyLine> objdumpParse(const QByteArray &output)
 {
     QVector<DisassemblyOutput::DisassemblyLine> disassemblyLines;
 
     QTextStream stream(output);
     QString asmLine;
+    const QRegularExpression disassemblyRegex(QStringLiteral("^   [0-9a-f]{4,}:       [0-9a-f ]{2,}"));
```

Comment:
`static`, and aren't you missing `<>` before the `:`?

Reply:
this is only to check if the line is an assembly line like this: `   4f616:       84 c0                   test   %al,%al` and not for the function header part.

Reply:
still, the regex should be static, and ideally you add what you wrote above as a comment:

```
// detect lines like:
// 4f616: 84 c0 test %al,%al
static const disassemblyRegex = QRegularExpression(QStringLiteral(...));
```